integration_test/src/main.rs

```diff
@@ -2,6 +2,272 @@
 //!
 //! This is how some external user would use rust-miniscript
 
+extern crate bitcoincore_rpc;
+extern crate log;
+
+extern crate bitcoin;
+extern crate miniscript;
+
+use bitcoincore_rpc::{json, Auth, Client, RpcApi};
+
+use bitcoin::secp256k1;
+use bitcoin::util::bip143;
+use bitcoin::util::psbt;
+use bitcoin::util::psbt::PartiallySignedTransaction as Psbt;
+use bitcoin::{Amount, OutPoint, Transaction, TxIn, TxOut, Txid};
+mod read_file;
+use miniscript::miniscript::iter;
+use miniscript::DescriptorTrait;
+use miniscript::MiniscriptKey;
+use miniscript::{Miniscript, Segwitv0};
+use std::collections::BTreeMap;
+
+struct StdLogger;
+
+impl log::Log for StdLogger {
+    fn enabled(&self, metadata: &log::Metadata) -> bool {
+        metadata.target().contains("jsonrpc") || metadata.target().contains("bitcoincore_rpc")
+    }
+
+    fn log(&self, record: &log::Record) {
+        if self.enabled(record.metadata()) {
+            println!(
+                "[{}][{}]: {}",
+                record.level(),
+                record.metadata().target(),
+                record.args()
+            );
+        }
+    }
+
+    fn flush(&self) {}
+}
+
+static LOGGER: StdLogger = StdLogger;
+
+static mut VERSION: usize = 0;
+/// Get the version of the node that is running.
+fn version() -> usize {
+    unsafe { VERSION }
+}
```

Comment:
As near as I can tell there is no reason for this function or the `static mut` global to exist